File: src/main/java/com/samsad/reactbackend/entity/Note.java
package com.samsad.reactbackend.entity;

import lombok.Data;
import javax.persistence.*;

@Entity
@Table(name = "tbl_notes")
@Data
public class Note {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    Long id;
    String title;
    String body;
    String category;
}

File: src/main/kotlin/com/samsad/springbootnotes/entity/Note.kt
package com.samsad.springbootnotes.entity

import lombok.Data
import javax.persistence.*

@Entity
@Table(name = "tbl_notes")
@Data
data class Note(
        @Id
        @GeneratedValue(strategy = GenerationType.IDENTITY)
        val id: Long? = null,
        val title: String? = null,
        val body: String? = null,
        val category: String? = null
)